app/Resources/views/page.html.twig

```diff
@@ -226,7 +226,7 @@
         </noscript>
     {% endif %}
 
-    <div class="global-wrapper" data-behaviour="FragmentHandler Math{% if hypothesis ?? false %} HypothesisLoader{% endif %}"
+    <div class="global-wrapper" data-behaviour="{% if cookie_consent %} CookieOverlay {% endif %}FragmentHandler Math{% if hypothesis ?? false %} HypothesisLoader{% endif %}"
```

Comment:
Is there any reason for this to be here if Cookiebot is handling everything automatically?